### ISSUE TITLE
Can not locate EEZ offshore wind shape file 

### ISSUE THREAD

**Issue by trevorb1** [PyPSA/pypsa-usa#139]
### Checklist

- [X] I am using the current [`master`](https://github.com/PyPSA/pypsa-usa/tree/master) branch
- [X] I am running on an up-to-date [`pypsa-usa` environment](https://github.com/PyPSA/pypsa-usa/blob/master/workflow/envs/environment.yaml). Update via `conda env update -f envs/environment.yaml`

### The Issue

When I try to run the workflow with default configuration I get the following error: 

```bash
MissingInputException in rule build_shapes in file /home/trevor/repos/pypsa-usa/workflow/Snakefile, line 243:
Missing input files for rule build_shapes:
    output: resources/texas/country_shapes.geojson, resources/texas/onshore_shapes.geojson, resources/texas/offshore_shapes.geojson, resources/texas/state_boundaries.geojson
    wildcards: interconnect=texas
    affected files:
        data/gis/conus_eez.shp
```

As the CA_OSW shapes are still in the `repo_data/` folder, did the EEZ shapefile just got missed being commited, @ktehranchi?  However, the snakefile says it should be under the `data/` folder (which should pull from Zenodo), but I don't see `gis/conus_eez.shp` in the [deposit](https://zenodo.org/records/10067222) either? 

### Steps To Reproduce

1. Clone the repo 
2. Run `snakemake -j6`

### Expected Behavior

The workflow should run 

### Anything else?

Would it be worth moving the following configuration option to the `config.osw.yaml` file? 

```bash
offshore_shape:
  use: eez #options are ca_osw, eez
```

**Comment by trevorb1** [comment]
Also, the final else statement where the `offshore_config` is used I believe should read 

```python 
    else:
        logger.error(f"source {offshore_config} is invalid offshore data source")
        offshore = None
```

(ie. replace the `{source}` with `{offshore_config}` in the f string) 

https://github.com/PyPSA/pypsa-usa/blob/2c1833191619828847824ddc43b596c249e08cc9/workflow/scripts/build_shapes.py#L253-L262

**Comment by trevorb1** [comment]
The f string issue has been fixed in PR #141
